### PR TITLE
refactor: restore command handlers

### PR DESCRIPTION
**Pull request by jwartofsky-yext**
Use command handlers that were used previously with Yargs.

As opposed to action written directly in command.

This also fixes 'useProdUrls' in dev.ts

Tested all commands and ran auto tests.

### CODE REVIEW

File: packages/pages/src/dev/dev.ts [hunk 1]
```diff
@@ -4,6 +4,39 @@ import runSubProcess from "../util/runSubprocess.js";
...
+  }
+  if (!noGenFeatures)
+    await runSubProcess(
+      "pages generate features",
+      scope ? ["--scope" + " " + scope] : []
+    );
+
+  await createServer(!local, !!useProdURLs, scope);
+
+  if (openBrowser) await open(`http://localhost:${devServerPort}/`);
```
Comment (benlife5) [inline packages/pages/src/dev/dev.ts:38]: The openBrowser argument still seems to not be passed through correctly

[JIRA ticket](https://yexttest.atlassian.net/browse/SUMO-5190?atlOrigin=eyJpIjoiYjQ5MTI3YjM1ZDAyNGYzODk4NDFmMWJmOTBkNTdiYjYiLCJwIjoiaiJ9)

**Review by benlife5 — COMMENTED** [review]
Should also update prod.ts

File: packages/pages/src/dev/dev.ts (hunk 1 above)
Comment (jwartofsky-yext) [reply packages/pages/src/dev/dev.ts:38]: The openBrowser argument works. When running with npm run dev, the argument has to be passed to the inner command 'pages dev'. 

You can do so like this: `npm run dev -- --no-open-browser`

**Review by mkilpatrick — COMMENTED** [review]
Update type and remove the scope to `refactor: restore...`

plugin refers to our Vite plugin, aka the prod build. You are just refactoring the binary commands which we don't have a scope for.

**Review by benlife5 — APPROVED** [review]

**Pull request merged**